migrations/sqls/20240814102304-alter-licence-version-created-and-updated-at-down.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE IF EXISTS water.licence_versions ALTER COLUMN date_created SET NOT NULL;
+ALTER TABLE IF EXISTS water.licence_versions ALTER COLUMN date_updated SET NOT NULL;
```

Comment:
This is not my understanding on how to remove a default from a column. I think instead it's supposed to be

```suggestion
ALTER TABLE IF EXISTS water.licence_versions ALTER COLUMN date_created DROP DEFAULT;
ALTER TABLE IF EXISTS water.licence_versions ALTER COLUMN date_updated DROP DEFAULT;
```

Having run the migration, if you run the migration down command locally you should see the column properties revert to what they were before the migration was applied.